.travis.yml

```diff
@@ -3,7 +3,11 @@ env:
   - GO111MODULE=on
 go:
   - 1.11.x
+before_script:
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.13.2
 script:
+  - golangci-lint run
   - go test -v -race  -coverprofile=coverage.txt -covermode=atomic ./...
+  - go tool cover -func=coverage.txt | awk '/^total:/ { gsub(/%$/, ""); if ($3 < 100.0) {print "FAIL\x3A Coverage below 100%"; exit(1)} }'
```

Comment:
Not sure why the awk print is using `\x3A` for a colon `:`. There is a literal colon in the awk pattern, so yaml doesn't care. awk doesn't care either.